lib/llog/api.go

```diff
@@ -0,0 +1,159 @@
+package llog
+
+import (
+	"io"
+)
+
+// Level indicates logging level. The higher the value, the less important.
+// Any non-negative integer is always less important than a named constant.
```

Comment:
I think it's better to have the most important level be the smallest number (0). This is more natural and also makes it easier to implement debug levels.